pkgs/top-level/python-packages.nix

```diff
@@ -6053,6 +6053,8 @@ in {
 
   notebook = callPackage ../development/python-modules/notebook { };
 
+  notebook_shim = callPackage ../development/python-modules/notebook_shim { };
```

Comment:
```suggestion
  notebook-shim = callPackage ../development/python-modules/notebook-shim { };
```

Reply:
I am curious about the rule to name the attribute. I was under the impression that the same name used by `pip install` should be used as attribute (in this case `notebook_shim`).

Reply:
No, they are normalized https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md?plain=1#L1718-L1723

Reply:
ah okay. it thought i should stick to the pypi-package-name